src/InputPicker/InputPicker.tsx

```diff
@@ -668,7 +683,12 @@ const InputPicker: PickerComponent<InputPickerProps> = React.forwardRef(
 
       let items: ItemDataType[] = filterNodesOfTree(getAllData(), checkShouldDisplay);
 
-      if (creatable && searchKeyword && !items.find(item => item[valueKey] === searchKeyword)) {
+      if (
+        creatable &&
+        (typeof shouldDisplayCreateOption === 'function'
+          ? shouldDisplayCreateOption(searchKeyword, items)
+          : searchKeyword && !items.find(item => item[valueKey] === searchKeyword))
```

Comment:
```diff
- items.find(item => item[valueKey] === searchKeyword)
+ items.find(item => item[valueKey] === searchKeyword || item[labelKey] === searchKeyword)
```
If adding a matching check of [labelKey] can solve the problem in the current scenario?

Reply:
This would break the current behavior, where users might expect not hiding items with a matching `labelKey`.